### ISSUE TITLE
[TW-1399] Needless space after opening parentheses 

### ISSUE THREAD

**Issue by taskwarrior** [GothenburgBitFactory/taskwarrior#1426]
_Peter Vitt on 2014-08-22T15:10:40Z says:_ 

 When I add a opening parentheses into an annotation via '\(', a needless space is added after it. 

Example: '\(This goes into parentheses\)' results in '( This goes into parentheses)'.

It is not a major bug, but it disturbs my reading flow.

**Comment by taskwarrior** [comment]
Migrated metadata:
```
Created: 2014-08-22T15:10:40Z
Modified: 2014-08-23T16:15:51Z
```

**Comment by taskwarrior** [comment]
_Paul Beckingham on 2014-08-23T16:15:51Z says:_ 

 Thank you for reporting this.

While this is already fixed in the 2.4.0 branch, I want to encourage folks to report all the command line parser craziness they see, just to make sure we didn't miss any.
